roles/openshift_facts/library/openshift_facts.py

```diff
@@ -608,7 +608,11 @@ def build_controller_args(facts):
     cloud_cfg_path = os.path.join(facts['common']['config_base'],
                                   'cloudprovider')
     if 'master' in facts:
-        controller_args = {}
+        controller_args = {
+            'pv-recycler-pod-template-filepath-nfs': ['/etc/origin/master/recycler_pod.yaml'],
+            'pv-recycler-pod-template-filepath-hostpath': ['/etc/origin/master/recycler_pod.yaml']
```

Comment:
We always set these? I'd just add them to the master-config.yaml template. If they need to be added during upgrade use yedit module to add the keys to master-config.yaml

We should try to avoid updating openshift_facts/library/openshift_facts.py if possible.

Reply:
There are some examples here https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_control_plane/tasks/upgrade.yml#L59-L64

Reply:
fixed.